apps/etterlatte-testdata/src/main/kotlin/no/nav/etterlatte/testdata/features/dolly/DollyFeature.kt

```diff
@@ -151,7 +160,7 @@ class DollyFeature(
                     val soeker =
                         when (ytelse) {
                             // TODO
-                            SoeknadType.BARNEPENSJON -> nySoeknadRequest.barn.first()
+                            SoeknadType.BARNEPENSJON -> barnListe.first()
```

Comment:
Burde søker være et eget parameter for å unngå at vi bare henter ut første fra listen over barn? Ser for meg at det de prøver på her er å søker for en spesifikk person?